articles/Frontend/Components/calculator.md

```diff
@@ -0,0 +1,843 @@
+# Calculator component overview
+
+In order to properly support the [mathematical order of operations](https://en.wikipedia.org/wiki/BODMAS)
+we designed another calculator component, able to work on expressions instead of
```

Comment:
I don't know what `another` refers to?

Reply:
I changed the intro, so the `another` now gets a context.